WootricSDK/WootricSDK/UIItems.m

```diff
@@ -24,9 +24,28 @@
 
 #import "UIItems.h"
 #import "WTRColor.h"
+#import "NSString+FontAwesome.h"
+#import <CoreText/CTFontManager.h>
 
 @implementation UIItems
 
++ (void)dinamycallyAddFont {
+    NSString *fontPath = [[NSBundle bundleForClass:[UIItems class]] pathForResource:@"fontawesome-webfont" ofType:@"ttf"];
+    NSData *fontData = [NSData dataWithContentsOfFile:fontPath];
+    
+    CFErrorRef error;
+    CGDataProviderRef provider = CGDataProviderCreateWithCFData((CFDataRef) fontData);
+    CGFontRef font = CGFontCreateWithDataProvider(provider);
+    if (!CTFontManagerRegisterGraphicsFont(font, &error)) {
+        if (CFErrorGetCode(error) == kCTFontManagerErrorAlreadyRegistered) return;
+        CFStringRef errorDescription = CFErrorCopyDescription(error);
+        NSLog(@"WootricSDK: Failed to load font: %@", errorDescription);
+        CFRelease(errorDescription);
+    }
+    CFRelease(font);
+    CFRelease(provider);
+}
```

Comment:
Correct, is for memory management. I'm responsible of releasing both objects for their memory to be deallocated and then the object destroyed.